router/router_test.go

```diff
@@ -561,37 +561,20 @@ func TestRouter(t *testing.T) {
 				req := httptest.NewRequest("GET", url, http.NoBody)
 				r.ServeHTTP(res, req)
 
-				Convey("Then no request is sent to the release calendar handler", func() {
-					So(len(releaseCalendarHandler.ServeHTTPCalls()), ShouldEqual, 0)
+				Convey("Then the request is sent to the babbage handler (the default) and no call to release calendar", func() {
+					So(len(babbageHandler.ServeHTTPCalls()), ShouldEqual, 1)
+					So(babbageHandler.ServeHTTPCalls()[0].In2.URL.Path, ShouldResemble, url)
```

Comment:
You haven't checked no call to the release calendar here - so it shouldn't be in the description, otherwise it's misleading. Up to you if you want to add it or remove the description.